src/components/Inputs/SearchBox/SearchBox.test.tsx

```diff
@@ -22,4 +22,10 @@ describe('SearchBox', () => {
         const wrapper = mount(<SearchBox />);
         expect(wrapper.containsMatchingElement(<SearchBox />)).toEqual(true);
     });
+    test('text input renders with partially customized icon button', () => {
```

Comment:
> partially customized icon button
what does this mean? what is partial about it?

Reply:
Like, only 1 property is set. Tests that the defaults work and the component doesn't cause and error.